mws/resource_mws_network_connectivity_config.go

```diff
@@ -0,0 +1,74 @@
+package mws
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/common"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func ResourceMwsNetworkConnectivityConfig() common.Resource {
+	s := common.StructToSchema(settings.NetworkConnectivityConfiguration{}, func(m map[string]*schema.Schema) map[string]*schema.Schema {
+		common.CustomizeSchemaPath(m, "name").SetValidateFunc(validation.StringLenBetween(3, 30))
+		for _, p := range []string{"name", "region"} {
+			common.CustomizeSchemaPath(m, p).SetRequired()
+			common.CustomizeSchemaPath(m, p).SetForceNew()
```

Comment:
You can chain them together: `.SetRequired().SetForceNew()`

Reply:
done